libraries/AP_Landing/AP_Landing_Deepstall.cpp

```diff
@@ -341,24 +364,38 @@ bool AP_Landing_Deepstall::override_servos(void)
         return false;
     }
 
-    SRV_Channel* elevator = SRV_Channels::get_channel_for(SRV_Channel::k_elevator);
-
-    if (elevator == nullptr) {
-        // deepstalls are impossible without these channels, abort the process
-        GCS_SEND_TEXT(MAV_SEVERITY_CRITICAL, "Deepstall: Unable to find the elevator channels");
```

Comment:
This is probably an important check; perhaps we could check for non-zero on SRV_Channel::k_elevator instead?